app/src/main/java/com/example/android/timepicker/MainActivity.java

```diff
@@ -13,13 +14,13 @@
     private TextView time;
     private Calendar calendar;
     private String format="";
-
+    private Toast toast;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        timePicker1=(TimePicker)findViewById(R.id.timePicker1);
-        time=(TextView)findViewById(R.id.textView1);
+        timePicker1=findViewById(R.id.timePicker1);
```

Comment:
Why you have removed the typecasting?

Reply:
**Type casting are no longer required,got a warning to remove them while commiting from studio**

![Screenshot from 2019-03-21 20-07-13](https://user-images.githubusercontent.com/29865288/54759947-190fc980-4c15-11e9-9adc-44a8acd71e94.png)